src/controller/CHIPDeviceController.cpp

```diff
@@ -629,7 +630,8 @@ CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, co
 CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, RendezvousParameters & params)
 {
     MATTER_TRACE_SCOPE("EstablishPASEConnection", "DeviceCommissioner");
-    CHIP_ERROR err                     = CHIP_NO_ERROR;
+
+    MATTER_LOG_METRIC_SCOPE_WITH_ERROR(chip::Tracing::kMetricPASESession, err, CHIP_NO_ERROR);
```

Comment:
you already have a scope above.this should probably remove that or re-use bits.

Reply:
This is resolved offline